changelog.d/280.added.md

```diff
@@ -0,0 +1 @@
+Log warning if secrets file is readable by other users than zino user
```

Comment:
This wording is not 100% in sync with the actual code. 

The code checks if the readable bit is set for OTHER users, it doesn't care who the zino user is.  It also doesn't care if there is some non-zino-user group that can access the file (but that would also complicate the logic).

IMHO, verifying that the file is not world-readable is sufficient for most cases, but the changelog might read as follows instead:

```suggestion
Log warning if `secrets` file is world-readable
```